packages/CodeGen/config.json

```diff
@@ -261,5 +261,12 @@
             "timeout": 30000,
             "when": "after"
         }
-    ]
+    ],
+    "metadataLogging" :{
```

Comment:
@JS-BC  I don't like the naming - call it `metadataSQLOutput` here and in the codebase around it for consistency. This is clearer as to what it actually is.